O/ONNXRuntime/build_tarballs.jl

```diff
@@ -0,0 +1,124 @@
+        install_license onnxruntime-$target/onnxruntime-*/LICENSE
+    else
+        # Cross-compiling for aarch64-apple-darwin on x86_64 requires setting arch.: https://github.com/microsoft/onnxruntime/blob/v1.10.0/cmake/CMakeLists.txt#L186
+        if [[ $target == aarch64-apple-darwin* ]]; then
+            cmake_extra_args="-DCMAKE_OSX_ARCHITECTURES='arm64'"
+        fi
+
+        cd onnxruntime
+        git submodule update --init --recursive
+        mkdir -p build
+        cd build
+        cmake $WORKSPACE/srcdir/onnxruntime/cmake \
+            -DCMAKE_INSTALL_PREFIX=$prefix \
+            -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DONNX_CUSTOM_PROTOC_EXECUTABLE=$host_bindir/protoc \
+            -Donnxruntime_BUILD_SHARED_LIB=ON \
+            -Donnxruntime_BUILD_UNIT_TESTS=OFF \
+            -Donnxruntime_DISABLE_RTTI=OFF \
+            $cmake_extra_args
+        make -j $nproc
+        make install
+        install_license $WORKSPACE/srcdir/onnxruntime/LICENSE
+    fi
+else
+    if [[ $target == *-w64-mingw32* ]]; then
+        chmod 755 onnxruntime-$target-cuda/onnxruntime-*/lib/*
+    fi
+    mkdir -p $includedir $libdir
+    cp -av onnxruntime-$target-cuda/onnxruntime-*/include/* $includedir
+    find onnxruntime-$target-cuda/onnxruntime-*/lib -not -type d | xargs -Isrc cp -av src $libdir
+    install_license onnxruntime-$target-cuda/onnxruntime-*/LICENSE
+fi
+
+if [[ $bb_full_target == aarch64-linux-gnu*cuda* ]]; then
+    cd $WORKSPACE/srcdir
+    unzip -d onnxruntime-$target-cuda onnxruntime-$target-cuda.whl
+    mkdir -p $libdir
+    find onnxruntime-$target-cuda/onnxruntime_gpu*.data/purelib/onnxruntime/capi -name *.so* -not -name *py* | xargs -Isrc cp -av src $libdir
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+function platform_exclude_filter(p::Platform)
+    libc(p) == "musl" ||
+    p == Platform("i686", "Linux") || # No binary - and source build fails linking CXX shared library libonnxruntime.so
+    Sys.isfreebsd(p)
+end
+platforms = supported_platforms(; exclude=platform_exclude_filter)
+platforms = expand_cxxstring_abis(platforms; skip=!Sys.islinux)
+
+cuda_platforms = Platform[]
+for p in [
+    Platform("x86_64", "Linux"; cuda = cuda_tag),
+    Platform("x86_64", "Windows"; cuda = cuda_tag)
+]
+    push!(platforms, p)
+    push!(cuda_platforms, p)
+end
+push!(platforms, Platform("aarch64", "Linux"; cuda = cuda_aarch64_tag))
+push!(cuda_platforms, Platform("aarch64", "Linux"; cuda = cuda_aarch64_tag))
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["libonnxruntime", "onnxruntime"], :libonnxruntime)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("CUDNN_jll", cudnn_version;
+        compat = cudnn_compat,
+        platforms = cuda_platforms),
```

Comment:
Why do you need to specify this?

Reply:
Ah - yes - you mean I could (of course) leave out CUDNN out as there is a dependency on TensorRT?

Reply:
I just meant the `platforms` kwargs.

Reply:
Ah, OK - this was just to only have this (and/or the TensorRT) dependency on the CUDA platforms.